apps/files/src/components/FileList.vue

```diff
@@ -85,63 +63,33 @@
         </oc-table>
       </div>
       <oc-grid gutter="large" class="uk-width-1-1 uk-padding-small" v-if="!loadingFolder">
-        <div v-if="activeFilesCount.folders > 0 || activeFilesCount.files > 0" class="uk-text-nowrap uk-text-meta">
-          <template v-if="activeFilesCount.folders > 0">
-            {{ activeFilesCount.folders }}
-            <translate :translate-n="activeFilesCount.folders" translate-plural="folders">folder</translate>
-          </template>
-          <translate v-if="activeFilesCount.folders > 0 && activeFilesCount.files > 0">and</translate>
-          <template v-if="activeFilesCount.files > 0">
-            {{ activeFilesCount.files }}
-            <translate :translate-n="activeFilesCount.files" translate-plural="files">file</translate>
-          </template>
-          <template v-if="activeFiles.length > 0">
-            &dash; {{ filesTotalSize | fileSize }}
-          </template>
-        </div>
-        <div v-if="!publicPage()" class="uk-visible@s uk-width-2-3 uk-width-1-2@xl uk-text-meta uk-flex" :class="{ 'uk-visible@xl' : _sidebarOpen  }">
-          <span class="uk-margin-small-right">
-            <translate>Used space:</translate> {{ quota.used | fileSize }}
-            <template v-if="quota.definition !== 'default' && quota.definition !== 'none'">
-              <translate>of</translate> {{ quota.definition }}
-            </template>
-          </span>
-          <div class="uk-width-expand oc-align-self-center" v-if="quota.definition !== 'default' && quota.definition !== 'none'">
-            <oc-progress :value="parseInt(quota.relative)" :max="100" class="uk-margin-remove" />
-          </div>
-        </div>
+        <slot name="footer" />
       </oc-grid>
     </div>
-    <oc-dialog-prompt name="change-file-dialog" :oc-active="changeFileName" v-model="newName" :ocError="changeFileErrorMessage"
-                      :ocTitle="_renameDialogTitle" ocConfirmId="oc-dialog-rename-confirm" @oc-confirm="changeName"
-                      @oc-cancel="changeFileName = false; newName = ''" />
-    <oc-dialog-prompt name="delete-file-confirmation-dialog" :oc-active="filesDeleteMessage !== ''"
-                      :oc-content="filesDeleteMessage" :oc-has-input="false" :ocTitle="_deleteDialogTitle"
-                      ocConfirmId="oc-dialog-delete-confirm" @oc-confirm="reallyDeleteFiles"
-                      @oc-cancel="setFilesDeleteMessage('')"/>
   </div>
 </template>
 <script>
-import OcDialogPrompt from './ocDialogPrompt.vue'
 import { mapGetters, mapActions, mapState } from 'vuex'
 
-import Mixins from '../mixins'
-
 export default {
-  components: {
-    OcDialogPrompt
-  },
-  mixins: [
-    Mixins
-  ],
   name: 'FileList',
-  props: ['fileData', 'starsEnabled', 'checkboxEnabled', 'dateEnabled', 'parentFolder'],
-  mounted () {
-    this.$_ocFilesFolder_getFolder()
+  props: {
```

Comment:
I'd feel safer if the props which are required would have it actually specified.